processing/src/main/java/io/druid/query/metadata/metadata/SegmentMetadataQuery.java

```diff
@@ -30,17 +31,43 @@
 import org.joda.time.Interval;
 
 import java.util.Arrays;
+import java.util.EnumSet;
 import java.util.Map;
 
 public class SegmentMetadataQuery extends BaseQuery<SegmentAnalysis>
 {
+  public enum AnalysisType
+  {
+    CARDINALITY,
+    SIZE;
+
+    @JsonValue
+    @Override
+    public String toString() {
```

Comment:
you technically don't have to override the toString method, you can create your own annotated method, e.g. `jsonString` in case you don't want to mess with the original toString